src/dmd/tocvdebug.d

```diff
@@ -916,317 +908,284 @@ void toDebugClosure(Symbol* closstru)
 
 int cvMember(Dsymbol s, ubyte *p)
 {
-    extern (C++) class CVMember : Visitor
+    scope v = new CVMember(p);
+    s.accept(v);
+    return v.result;
+}
+
+private extern (C++) class CVMember : Visitor
```

Comment:
why not move inside `cvMember`?

Reply:
This PR is moving it out. This has less indentation and has the body of `cvMember` all together.

Reply:
OK.